### DISCUSSION TITLE
Aruba CX switches using Netmiko

### DISCUSSION THREAD

**Discussion by arrvamsee** [ktbyers/netmiko#3477] [Q&A]
Hello All,

We have 2930s, 2530s, and CX6300s in our environment. I can configure the 2930s and 2530s without any problems using netmiko , but the **net_connect.send_config_set** command isn't working with the CX switches. However, **net_connect.send_command** works fine. 

The device_type used for CX switches is specified as 'device_type': 'aruba_os'.

Any help would be appreciated. 

Thank you

**Comment by ktbyers** [comment]
Try using the device_type:  `aruba_aoscx`

Let me know if that works.

**Reply by sungajjar** [reply]
Is aruba_aoscx added as a supported device_type?

**Reply by arrvamsee** [reply]
I remember trying it with device_type as aruba_CX but not aruba_aoscx. I will give it a try today and see if that works. 

Thank you

**Reply by arrvamsee** [reply]
I just gave it a try. device_type: aoscx and it did not work. Below is the error I am seeing

ValueError: Unsupported 'device_type' currently supported platforms are:
a10
accedian
adtran_os
adva_fsp150f2
adva_fsp150f3
alcatel_aos
alcatel_sros
allied_telesis_awplus
apresia_aeos
arista_eos
arris_cer
aruba_os
aruba_osswitch
aruba_procurve
audiocode_66
audiocode_72
audiocode_shell
avaya_ers
avaya_vsp
broadcom_icos
brocade_fastiron
brocade_fos
brocade_netiron
brocade_nos
brocade_vdx
brocade_vyos
calix_b6
casa_cmts
cdot_cros
centec_os
checkpoint_gaia
ciena_saos
cisco_asa
cisco_ftd
cisco_ios
cisco_nxos
cisco_s200
cisco_s300
cisco_tp
cisco_viptela
cisco_wlc
cisco_xe
cisco_xr
cloudgenix_ion
coriant
dell_dnos9
dell_force10
dell_isilon
dell_os10
dell_os6
dell_os9
dell_powerconnect
dell_sonic
dlink_ds
eltex
eltex_esr
endace
enterasys
ericsson_ipos
ericsson_mltn63
ericsson_mltn66
extreme
extreme_ers
extreme_exos
extreme_netiron
extreme_nos
extreme_slx
extreme_tierra
extreme_vdx
extreme_vsp
extreme_wing
f5_linux
f5_ltm
f5_tmsh
flexvnf
fortinet
generic
generic_termserver
hillstone_stoneos
hp_comware
hp_procurve
huawei
huawei_olt
huawei_smartax
huawei_vrp
huawei_vrpv8
ipinfusion_ocnos
juniper
juniper_junos
juniper_screenos
keymile
keymile_nos
linux
mellanox
mellanox_mlnxos
mikrotik_routeros
mikrotik_switchos
mrv_lx
mrv_optiswitch
netapp_cdot
netgear_prosafe
netscaler
nokia_srl
nokia_sros
oneaccess_oneos
ovs_linux
paloalto_panos
pluribus
quanta_mesh
rad_etx
raisecom_roap
ruckus_fastiron
ruijie_os
sixwind_os
sophos_sfos
supermicro_smis
teldat_cit
tplink_jetstream
ubiquiti_edge
ubiquiti_edgerouter
ubiquiti_edgeswitch
ubiquiti_unifiswitch
vyatta_vyos
vyos
watchguard_fireware
yamaha
zte_zxros
zyxel_os

**Reply by ktbyers** [reply]
Which version of Netmiko are you using. You must be using the latest released version of Netmiko (4.4.0)?

https://github.com/ktbyers/netmiko/releases/tag/v4.4.0

**Reply by sungajjar** [reply]
I upgraded to netmiko v4.4.0 and tested, it works fine. Thanks @ktbyers  👍
